wolvic/browser/vr/wvr_manager.h

```diff
@@ -16,6 +16,7 @@
 #include "mojo/public/cpp/bindings/pending_associated_remote.h"
 #include "mojo/public/cpp/bindings/receiver.h"
 #include "mojo/public/cpp/bindings/remote.h"
+#include "wolvic/browser/vr/moz_external_vr.h"
```

Comment:
can we keep this header in wvr_api's scope only?

Reply:
Not sure I get what you propose...